### ISSUE TITLE
Cookie bar, privacy policy and visitor counting with Clicky

### ISSUE THREAD

**Issue by phseiff** [phseiff/phseiff.github.io#7]
Add a bar asking for cookie acceptance, a privacy policy and embed the clicky visitor counter (but only load it when cookies are enabled).

**Comment by phseiff** [comment]
Cookie bar is now correctly formatted, sticky, and one can scroll down with it. I just need to add the button and text to the cookie bar, add a privacy policy, and add the functionality to agree to the cookie usage and to take ones agree back.

**Comment by phseiff** [comment]
Implemented and up and running. Looks great (better than I would've guessed before starting to work on it), and it also behaves like it's supposed to. I also added some suggestions for things I might want to implement regarding the privacy policy presentation sooner or later into the source code so I don't forget about them, but they are out of scope right now.
